src/components/UI/Card.js

```diff
@@ -1,12 +1,31 @@
-import React from "react";
+import React, {useState} from "react";
 import "./Card.css";
 
-function Card(props) {
+function Card({item}) {
+  let [classNames, setClassNames] = useState("card");
+
+  const clickHandler = () => {
+    if (classNames === "card") {
+      setClassNames("card card_selected");
+    } else {
+      setClassNames("card");
+    }
```

Comment:
It's not actually about the className value to be checked. In that way  if css class name is changed a bit - the whole logic should be overwritten. You should better semantically refer to whether card itself "is checked"/"is selected" or not.

So you can set the bool criteria isChecked as a state of the component.
`const [isChecked, setChecked] = useState(false);`
Then set the state itself as reversal to what it was before
`const clickHandler = () => setChecked(!isChecked);`
Then for card className you can check state in order to set the style:
`...<div className={"card" + isChecked ? " card_selected" : "" } > ... </div>`
Other way here - in order to not use styling concatenation you can use package [classnames](https://www.npmjs.com/package/classnames) for conditional styling.